src/sentry/static/sentry/app/views/organizationDiscover/utils.jsx

```diff
@@ -18,7 +18,7 @@ const VALID_QUERY_KEYS = [
 ];
 
 export function getQueryFromQueryString(queryString) {
-  const queryKeys = new Set([...VALID_QUERY_KEYS, 'utc']);
+  const queryKeys = new Set(VALID_QUERY_KEYS);
```

Comment:
I think this was needed to keep `utc` in the query string across discover queries

Reply:
`utc` isn't a valid value in a Discover querystring and would just be ignored anyway

Reply:
I think the date picker needs it though - I will take a look and remove `statsPeriod` and `utc` being required.